tests/race/BUILD.bazel

```diff
@@ -0,0 +1,21 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test", "go_binary")
+
+go_library(
+    name="go_default_library",
+    srcs = ["race.go"],
+    deps = ["//tests/race/racy:go_default_library"],
+)
+
+go_binary(
+    name = "race",
+    srcs = ["race_main.go"],
+    deps = [":go_default_library"],
+)
+
+go_test(
+    name = "go_default_test",
+    srcs = ["race_test.go"],
+    library = ":go_default_library",
+    size = "small",
+    tags = ["manual"],
```

Comment:
How will this get run in CI? Do we need a `sh_test` or a `bazel_test` that verifies this fails?